h5grove/utils.py

```diff
@@ -18,10 +19,27 @@ class LinkError(NotFoundError):
     pass
 
 
-def attr_metadata(attrId: h5py.h5a.AttrID) -> dict:
+def _get_attr_id(entity_attrs: h5py.AttributeManager, attr_name: str):
+    return entity_attrs.get_id(attr_name)
+
+
+def _legacy_get_attr_id(entity_attrs: h5py.AttributeManager, attr_name: str):
+    return h5py.h5a.open(entity_attrs._id, entity_attrs._e(attr_name))
+
+
+get_attr_id = (
+    _legacy_get_attr_id
+    if h5py_version.major <= 2 and h5py_version.minor <= 9
+    else _get_attr_id
```

Comment:
`get_id` was introduced in h5py 2.10: https://docs.h5py.org/en/stable/whatsnew/2.10.html